setup.py

```diff
@@ -84,9 +84,9 @@ def get_extensions():
     return extensions
 
 
-install_requires = []
+install_requires = ['torch']
 setup_requires = []
-tests_require = ['pytest', 'pytest-runner', 'pytest-cov']
+tests_require = ['pytest', 'pytest-cov', 'coverage']
```

Comment:
I think adding `coverage` here is unnecessary as this is only used in CI, not on a local dev machine. Happy to remove `pytest-runner` though :)

Reply:
Okay, fair point!